docs/index.rst

```diff
@@ -67,10 +67,13 @@ The following packages are required for astroquery installation & use:
 
 * `numpy <http://www.numpy.org>`_ >= 1.9
 * `astropy <http://www.astropy.org>`__ (>=1.0)
+* `astropy-healpix <http://astropy-healpix.readthedocs.io/en/latest/>`_
 * `requests <http://docs.python-requests.org/en/latest/>`_
 * `keyring <https://pypi.python.org/pypi/keyring>`_
 * `Beautiful Soup <https://www.crummy.com/software/BeautifulSoup/>`_
 * `html5lib <https://pypi.python.org/pypi/html5lib>`_
+* `regions <https://astropy-regions.readthedocs.io/en/latest/>`_
```

Comment:
these are still optional dependencies for the package, so the it was better the way you added before this commit.

Reply:
Yes, I will remove 9e0753e